File: Week06_Python/phonebook0.py
# Implements linear search for names using loop

# A list of names
names = ['Aqsa', 'Dita', 'Sofyan']

# Ask for name
name = input('Name: ')

# Search for name
for n in names:
    if n == name:
        print('Found')
        break
else:
    print('Not Found')

File: Week03_Algorithms/phonebook0.c
// Implements a phone book without structs

#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    // Arrays of strings
    string names[] = {"Dita", "Sofyan", "Aqsa"};
    string numbers[] = {"+62-123-456-1000", "+62-123-456-2000", "+62-123-456-3000"};

    // Search for name
    string name = get_string("Name: ");
    for (int i = 0; i < 3; i++)
    {
        if (strcmp(names[i], name) == 0)
        {
            printf("Found %s\n", numbers[i]);
            return 0;
        }
    }
    printf("Not found\n");
    return 1;
}